README.md

```diff
@@ -67,35 +67,45 @@ representation of Super JSON (a bit like Parquet).
 Even though SuperDB is based on these super-structured data formats, it can read and write
 most common data formats.
 
-## Try It
-
-Trying out SuperDB is super easy: just [install](https://superdb.org/docs/install)
-the command-line tool [`super`](https://superdb.org/docs/commands/super).
-
-Detailed documentation for the entire SuperDB system and its piped SQL syntax
-is available on the [SuperDB docs site](https://superdb.org/docs).
-
-The SuperDB query engine can run locally without a storage engine by accessing
-files, HTTP endpoints, or S3 paths using the `super` command. While
-[earlier in its development](https://superdb.org/docs/commands/super-db/#status),
-SuperDB can also run on a
-[super-structured data lake](https://superdb.org/docs/commands/super-db/#the-lake-model)
-using the `super db` sub-commands.
-
 ## Project Status
```

Comment:
I moved the **Project Status** section _above_ the **Try It** section since I figured it was good to draw their attention first to the project's early status.